components/Card.tsx

```diff
@@ -0,0 +1,17 @@
+
+const Card =()=>{
+    return(
+        <div className="card text-white bg-white bg-opacity-25 ">
+       <img className="card-img-top" src="/bella_fiori_mm.jpg" alt="Card image cap"></img>
```

Comment:
Use Next Image component for images

Reply:
The forementioned Bootstrap  pre-done component. Changing it all now. Again thanks on the remarks.